src/commands/generate-vdm.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+import { Command } from '@oclif/command';
+import { IBooleanFlag, IOptionFlag } from '@oclif/parser/lib/flags';
+import { generate, GeneratorOptions as GeneratorOptionsSDK, generatorOptionsCli as generatorOptionsSDK } from '@sap/cloud-sdk-generator';
+import { Options } from 'yargs';
+import { toBooleanFlag, toGeneratorSDK, toStringFlag } from '../utils/generate-vdm-util';
```

Comment:
The types exported here are used in the `generate-vdm-util`. This is a circular dependency (which can be resolved luckily). I would like it much more if the types could be moved out to a util file which has little to no imports and avoids this circular dependency completely.

Reply:
moved all types to the util class